addons/l10n_gcc_invoice/views/report_invoice.xml

```diff
@@ -456,7 +456,7 @@
                 <p t-if="o.narration" name="comment">
                     <div class="row">
                         <div class="col-6 text-start">
-                            <span t-out="o.narration"/>
+                            <span t-if="o._l10n_gcc_invoice_english_different_than_arabic('narration')" t-out="o.narration"/>
```

Comment:
Isn't this more readable and has the advantage of not needing a function?
```suggestion
                            <span t-if="o.narration != o_sec.narration" t-out="o.narration"/>
```

Reply:
@william-andre I don't really know what fields can be modified and translated by the user. If it's the case, it would make sense to do the same for them.

After searching, the fiscal position notes can also be modified so I added the same condition in the commit.

Reply:
Well it looks like it's the same for `invoice_payment_term_id.note` for instance; it should be quick to look for it

Reply:
Done :)